packages/dds/tree/src/tableSchema.ts

```diff
@@ -852,6 +852,19 @@ export namespace System_TableSchema {
 /**
  * Contains types and factories for creating schema to represent dynamic tabular data.
  *
+ * @remarks
+ *
+ * Tables created using these APIs are...
+ *
+ * - sparse, meaning that they only store the cells that are actually present in the table.
+ *
+ * - dynamic, meaning that their structure can be modified at runtime.
+ * Columns and rows can be inserted, removed, modified, and reordered.
+ * Cells can be inserted, removed, and modified.
+ *
```

Comment:
I think we could add some more items here, maybe:

```suggestion
 *
 * - row major, meaning that operating on rows (such as inserts, removal, moves, and traversal) is more efficient than operating on columns.
 *
 * leaky, meaning that rows can end up contain cells for columns which have been removed from the table.
```

Reply:
Good suggestions, thanks! I'm going to format these a bit different, but I'll add something to these effects.

Reply:
I captured the current state of things. Let me know what you think of the updates.